src/inverseRobot.js

```diff
@@ -3,11 +3,25 @@
 /*
  * @param {object} robot
  *
- * @return {object}
+ * @return {object | null}
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const fixed = {};
+  const valuesSet = new Set();
+
+  for (const key in robot) {
+    const value = robot[key];
```

Comment:
you are iterating through object in a for in loop, it is not the best way to go, especially if the object is large. Consider using `Object.keys()` or `Object.values()` to make the loop more efficient.